src/test/java/io/vertx/test/core/HttpTest.java

```diff
@@ -3813,6 +3813,44 @@ private void testFoo(String location, String expected) throws Exception {
     }
   }
 
+  @Test
+  public void testFollowRedirectEncodedUrl() throws Exception {
+    String value1 = "한글", value2 = "A B+C";
```

Comment:
`value1` should be escaped, i.e with \u00 , we should also do one value with the Euro sign as it's a 3 digits UTF8 encoded value

Reply:
Done